tensorboard/webapp/widgets/histogram/histogram_card_fob_test.ts

```diff
@@ -163,6 +163,7 @@ describe('HistogramCardFobController', () => {
       const fakeEvent = new MouseEvent('mousemove', {
```

Comment:
also some buttons references here that can be removed.

Reply:
Done.